scripts/indexjanitor.js

```diff
@@ -0,0 +1,50 @@
+(async () => {
+  const { Client } = require('@elastic/elasticsearch');
+
+  const client = new Client({
+    node: {
+      url: new URL(process.env.ELASTIC_URL),
+    },
+    auth: {
+      username: process.env.ELASTIC_USERNAME,
+      password: process.env.ELASTIC_PASSWORD,
+    },
+    maxRetries: 5,
+    requestTimeout: 60000,
+  });
+
+  const aliasesToCareAbout = ['site-search-kb', 'site-search-dev-blog', 'site-search-xe-test'];
+
+  const dryRun = process.env.DRY_RUN ? process.env.DRY_RUN === 'false' : true;
+  console.log({ dryRun });
+
+  const indices = await client.indices.get({ index: 'site-search-*' });
+  const indicesToClose = [];
+  const safeIndices = [];
+
+  for (const alias of aliasesToCareAbout) {
+    const aliasMeta = await client.indices.getAlias({ name: alias });
+    const indexName = Object.keys(aliasMeta)[0];
+    safeIndices.push(indexName);
```

Comment:
I know we're not using multiple indices per alias now, but we might as well account for it.

```suggestion
    const indexNames = Object.keys(aliasMeta);
    safeIndices.push(...indexNames);
```